File: src/main/groovy/net/zomis/brainf/model/ast/tree/CommentSyntax.java
package net.zomis.brainf.model.ast.tree;

import net.zomis.brainf.model.BrainfuckRunner;
import net.zomis.brainf.model.ast.CommentToken;

public class CommentSyntax extends Syntax {

    private final CommentToken token;

    public CommentSyntax(CommentToken token) {
        this.token = token;
    }

    @Override
    public void perform(BrainfuckRunner runner) {

    }

}

File: brainduck-kotlin/src/commonMain/kotlin/net/zomis/brainf/model/ast/tree/CommentSyntax.kt
package net.zomis.brainf.model.ast.tree;

import net.zomis.brainf.model.BrainfuckRunner;
import net.zomis.brainf.model.ast.CommentToken;

class CommentSyntax(private val token: CommentToken) : Syntax() {

    override fun perform(runner: BrainfuckRunner) {
    }

    override fun toString(): String {
        return "CommentSyntax{" + token.text + '}';
    }

}
